modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo.tsx

```diff
@@ -12,202 +12,494 @@
+
+	const millitaryAffiliationOptions = [
+		{
+			label: '',
+			value: '1',
+		},
+		{
+			label: 'Choose an option',
+			value: '2',
+		},
+		{
+			label: 'None',
+			value: '3',
+		},
+		{
+			label: 'US Air Force',
+			value: '4',
+		},
+		{
+			label: 'US Coast Guard',
+			value: '5',
+		},
+		{
+			label: 'US Marine Corps',
+			value: '6',
+		},
+		{
+			label: 'US Military',
+			value: '7',
+		},
+		{
+			label: 'US Navy',
+			value: '8',
+		},
+	];
 
-	const options = [
+	const highestEducationOptions = [
 		{
 			label: '',
 			value: '1',
 		},
 		{
-			label: 'CHOOSE AN OPTION',
+			label: 'Choose an option',
 			value: '2',
 		},
 		{
-			label: 'CA',
+			label: 'Hign School',
 			value: '3',
 		},
 		{
-			label: 'NV',
+			label: 'College',
 			value: '4',
 		},
 		{
-			label: 'NY',
+			label: 'Masters',
 			value: '5',
 		},
+		{
+			label: 'PHD',
+			value: '6',
+		},
+	];
+
+	const governmentAffiliationOptions = [
+		{
+			label: '',
+			value: '1',
+		},
+		{
+			label: 'Choose an option',
+			value: '2',
+		},
 	];
 
 	return (
-		<div className="mx-8">
-			<div className="font-weight-bolder text-paragraph-sm">
-				CONTACT INFORMATION
+		<div className="bg-neutral-0 mx-8">
+			<div className="px-3">
+				<h5 className="p-0 text-uppercase">{driver}</h5>
+
+				<hr className="mb-5"></hr>
+			</div>
+
+			<div className="font-weight-bolder text-paragraph-sm text-uppercase">
+				Driver Information
 			</div>
 
 			<hr></hr>
 
 			<ClayForm>
-				<div className="d-flex flex-row">
+				<div className="row">
 					<div className="col filled form-condensed form-group">
 						<ClayInput
-							className="bg-neutral-0"
+							autoComplete="off"
+							className="bg-neutral-0 w-100 year"
 							id="firstName"
 							name="firstName"
+							onChange={(event) => {
+								handleChangeField('year', event.target.value);
+							}}
 							required
 							type="text"
+							value={form.firstName}
 						/>
 
-						<label htmlFor="firstName">First Name*</label>
+						<label htmlFor="firstName">
+							Fisrt Name&nbsp;
+							<span className="text-danger-darken-1">*</span>
+						</label>
 					</div>
 
 					<div className="col filled form-condensed form-group">
 						<ClayInput
-							className="bg-neutral-0"
+							autoComplete="off"
+							className="bg-neutral-0 make w-100"
 							id="lastName"
 							name="lastName"
+							onChange={(event) => {
+								handleChangeField(
+									'lastName',
+									event.target.value
+								);
+							}}
 							required
 							type="text"
+							value={form.lastName}
 						/>
 
-						<label htmlFor="lastName">Last Name*</label>
+						<label htmlFor="lastName">
+							Last Name&nbsp;
+							<span className="text-danger-darken-1">*</span>
+						</label>
+					</div>
+
+					<div className="col filled form-condensed form-group">
+						<ClaySelect
+							aria-label="Select Label"
+							id="relationToContact"
+						>
+							{relationToContactOptions.map((item) => (
```

Comment:
Please, change all .map parameter names.

```suggestion
							{relationToContactOptions.map((relationToContactOption) => (
```

Reply:
Done!